server/assets-v1/templates/src/pages/client_portal/register.html

```diff
@@ -190,8 +190,15 @@ <h1 class="font-bold text-3xl md:text-4xl text-[#3751FE] mb-11 animate-slideFrom
               password: password.value,
             }),
           });
-          showToastMessage("Registration successful!", "success")
-          window.location.href = "[[ .ProxyURL ]]/client-login-page";
+          const registerResponseBody = await resp.json();
+          if (registerResponseBody.message === "OK!") {
+            showToastMessage(registerResponseBody.message, "success");
+            window.location.href = "[[ .ProxyURL ]]/client-login-page";
+          } else if (registerResponseBody.message) {
+            showToastMessage(registerResponseBody.message, "error");
+          } else {
+            showToastMessage("Something went wrong!", "error");
+          }
```

Comment:
Instead of doing the `if (registerResponseBody.message === "OK!")`

What if we do like this:
```suggestion
          if (registerResponseBody.status === false) {
             showToastMessage("Something went wrong!", "error");
            window.location.href = "[[ .ProxyURL ]]/client-login-page";
          } else {
            showToastMessage(registerResponseBody.message, "success");
            window.location.href = "[[ .ProxyURL ]]/client-login-page";
          }
```

In this way, we can get rid of the `message` field also as @stravid87  has commented, @NJavokhir  don't do this change for now it is just a suggestion, maybe you can do it afterwards in upcoming PRs and do some testing on it.